submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,29 @@
+## A scientific examination of the Factorio world
+
+Suspension of disbelief is a well-known concept, however that does not stop me from ignoring it completely. Some days ago I saw a [Reddit post](https://www.reddit.com/r/factorio/comments/jaz6yl/calculating_the_density_of_nauvis/) about calculating the density of Nauvis, and that gave me an idea of making a just-for-fun compilation of all the ridiculous science that goes on in the Factorio universe. I will not go into the formulas because that would make this article sound like a research paper. Instead, I leave the exact calculations as an exercise for the especially dedicated reader.
+
+### Nauvis is a white dwarf
+
+This is a short recap of the Reddit post by [u/DaveMcW](https://www.reddit.com/user/DaveMcW/) that I just mentioned. We know the length of one day/night cycle on the planet, which gives us its angular velocity. By stating that the centripetal force from the planet's rotation has to be less than gravity at the equator and then cancelling a bunch of values, we get that Nauvis's density has to be greater than 813400 kg/m\^3. That makes Nauvis at least as dense as a white dwarf star.
+
+### A bit about Nauvis's orbital mechanics
+
+Since the shadows are at a constant length and direction throughout the day, the only possible explanation is that the light source doesn't move, meaning Nauvis is tidally locked. This means the day/night cycle is caused by a different object occluding the light. Based on the frequency of the nights, that object is probably in orbit around Nauvis, and it can't be a moon, since a moon large enough for such long nights would tidally lock Nauvis to itself. So night must be caused by a cloud of debris in orbit of Nauvis, which also gives a reason for the Engineer crashing there in the first place.
```

Comment:
This is good but could maybe be stretched out a bit more to lower the information density? Could do with a bit more intro and maybe an image too of the orbital arrangements of the planets? (could just be a paint mockup to illustrate it lol)